creator-node/src/components/tracks/trackContentUploadManager.js

```diff
@@ -264,28 +261,23 @@ function createSegmentToDurationMap({
 }
 
 /**
- * Save transcode and segment files (in parallel batches) to ipfs and copy to disk.
+ * Save transcode and segment files (in parallel batches) to disk.
  * @param {Object} batchParams
- * @param {string} batchParams.cnodeUserUUID the observed user's uuid
  * @param {string} batchParams.fileDir the dir path of the temp track artifacts
  * @param {Object} batchParams.logContext
  * @param {string} batchParams.transcodeFilePath the transcoded track path
  * @param {string} batchParams.segmentFilePaths the segments path
  * @returns an object of array of segment multihashes, src paths, and dest paths and transcode multihash and path
  */
 async function batchSaveFileToIPFSAndCopyFromFS({
-  cnodeUserUUID,
   fileDir,
   logContext,
   transcodeFilePath,
   segmentFileNames
 }) {
-  const multihash = await FileManager.saveFileToIPFSFromFS(
```

Comment:
interesting - `saveFileToIPFSFromFS()` would copy the file on disk to `/multihash/CID` but the below step `FileManager.copyMultihashToFs()` already does that...
so we were just copying it twice?

Reply:
there was a flag to do copying or to only generate a hash -- I think it wasn't copying the first time because it was doing the only-hash logic even though the function name makes it sound like it copies every time

Reply:
ah yeah, cool